packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSlider.tsx

```diff
@@ -92,26 +116,84 @@ export const AnonymityLevelSlider = ({ className }: AnonymityLevelSliderProps) =
 
     const isErrorDisplayed = anonymityStatus === AnonymityStatus.Bad;
 
+    const labels = useMemo(
+        () => [
+            {
+                value: 1,
+                component: (
+                    <Label>
+                        <Icon icon="ONE_USER" size={14} color={theme.TYPE_DARK_GREY} /> 1
```

Comment:
This also trigger DOM nesting warning, Icon is wrapped in `div` and Range Label is `p`

Reply:
6b84fc60c0782730d227ef3e2f51fb09ea1a7b56